app/serializers/credit_card_serializer.rb

```diff
@@ -1,3 +1,3 @@
 class CreditCardSerializer < ActiveModel::Serializer
-  attributes :id, :month, :year, :number, :first_name, :last_name
+  attributes :id, :month, :year, :card_number, :first_name, :last_name, :cvv, :last_four
```

Comment:
@ksolkowski's point isn't valid, what if the CC fails to save, the number would still be there and the user should be able to edit it in the form